ext/node/ops/sqlite/statement.rs

```diff
@@ -8,22 +8,57 @@ use std::rc::Weak;
 use deno_core::op2;
 use deno_core::v8;
 use deno_core::v8::GetPropertyNamesArgs;
+use deno_core::v8_static_strings;
 use deno_core::GarbageCollected;
 use rusqlite::ffi;
-use serde::Serialize;
 
+use super::validators;
 use super::SqliteError;
 
 // ECMA-262, 15th edition, 21.1.2.6. Number.MAX_SAFE_INTEGER (2^53-1)
 const MAX_SAFE_JS_INTEGER: i64 = 9007199254740991;
 
-#[derive(Serialize)]
-#[serde(rename_all = "camelCase")]
 pub struct RunStatementResult {
   last_insert_rowid: i64,
   changes: u64,
 }
 
+impl RunStatementResult {
+  fn serialize<'a>(
```

Comment:
Ditto: is this some magical method as a `serde_v8` replacemant?

Reply:
updated to `ToV8`